pages.it/windows/xcopy.md

```diff
@@ -0,0 +1,36 @@
+# xcopy
+
+> Copia di file e directory.
+> Ulteriori informazioni: <https://learn.microsoft.com/windows-server/administration/windows-commands/xcopy>.
```

Comment:
Thanks for contribution. There is a template for "More information", see https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py#L24 .
```suggestion
> Maggiori informazioni: <https://learn.microsoft.com/windows-server/administration/windows-commands/xcopy>.
```